diff -Nurwa radare-1.0b-v/gui/Makefile radare-1.0b/gui/Makefile
--- radare-1.0b-v/gui/Makefile	2008-07-21 12:27:56.000000000 +0200
+++ radare-1.0b/gui/Makefile	2008-11-08 15:46:56.921804588 +0100
@@ -27,12 +27,12 @@
 
 install:
 	${E} 'Installing gradare'
-	${Q}${INSTALL_DIR} ${PREFIX}/bin
-	${Q}${INSTALL_PROGRAM} ${BIN} ${PREFIX}/bin
-	-${Q}rm -rf ${DATADIR}/radare/gradare/*
-	${Q}mkdir -p ${DATADIR}/radare/gradare/
-	${Q}cp -rf grsc/* ${DATADIR}/radare/gradare/
-	-${Q}chmod +x ${DATADIR}/radare/gradare/Shell
+	${Q}${INSTALL_DIR} ${DESTDIR}${PREFIX}/bin
+	${Q}${INSTALL_PROGRAM} ${BIN} ${DESTDIR}${PREFIX}/bin
+	-${Q}rm -rf ${DESTDIR}${DATADIR}/radare/gradare/*
+	${Q}mkdir -p ${DESTDIR}${DATADIR}/radare/gradare/
+	${Q}cp -rf grsc/* ${DESTDIR}${DATADIR}/radare/gradare/
+	-${Q}chmod +x ${DESTDIR}${DATADIR}/radare/gradare/Shell
 
 clean:
 	${E} 'Cleaning gui'
